src/components/chat-sidebar/chat-sidebar-link.tsx

```diff
@@ -1,31 +1,36 @@
-import { Star } from "lucide-react";
+import { Trash2 } from "lucide-react";
 import Link from "next/link";
 
 import { Button } from "@/components/ui/button";
 import { SidebarMenuButton, SidebarMenuItem } from "@/components/ui/sidebar";
+import { useChats } from "@/hooks/use-chats";
 
 export function ChatSidebarLink({
-  href,
+  uuid,
   title,
-}: Readonly<{ href: string; title: string }>) {
+}: Readonly<{ uuid: string; title: string }>) {
+  const chats = useChats();
   return (
     <SidebarMenuItem>
       <SidebarMenuButton asChild>
-        <Link href={href}>
+        <Link
+          href={`chats/${uuid}`}
```

Comment:
**issue (blocking):** A slash is required at the beginning of a `href` if we don't want it to be relative to the current path. Currently, this link redirects to `/chat/chats/<uuid>` because the sidebar links are only accessible on the `/chat` route - this is not the behavior we want as this gets us a 404 - there's no such path in our app. Changing this to the code below fixes this issue
```tsx
<Link
  href={`/chat/${uuid}`}
  className="flex flex-row items-center justify-between"
>
```

Reply:
oops, I forgot about it, will fix it rlq